recipes/pylint-plugin-utils/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pylint-plugin-utils" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "6844b4d0525aa41ff69c5d44c6edc640e3dab98446abeb499501757f09a3cd71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  {% set pypi_name = name.replace("_", "-") %}
```

Comment:
I propose that we keep variable setting up at the top, just for easier locatability.  I see why you moved it down in this particular case.  Not a big deal if you disagree.  I'm not terribly strong-feeling on it.